application/views/scripts/lan/info.phtml

```diff
@@ -19,79 +19,75 @@ if (is_null($lan->getRegistrationEndDateTime())) {
 <div id="view-content">
     <?php
 
-if ($lan != null) {
-    $participants = $this->participants;
-    ?>
-    <div id="title">
-        <h1>Informationen <?= $this->escape($lan->getName()) ?></h1>
-    </div>
-    <div id="lans">
-        <hr />
-        <p id="participate">
-            <?php
-    
-echo $this->partial('partials/lan/buttons.phtml', 
-            array(
-                    'lanId' => $lan->getId(),
-                    'hasPaid' => $hasPaid,
-                    'hasTicket' => $hasTicket,
-                    'registrationPossible' => $registrationPossible
-            ));
-    ?>
-        </p>
-        <div id="infos">
-            <?php if ($infos != null) { ?>
-                <?php foreach ($infos as $info) { ?>
-            <hr />
-            <h3><?php echo $this->escape($info->getName()) ?></h3>
-            <p class="lan-info"><?php echo $info->getInfo() ?></p>
-                <?php } ?>
-            <?php } else { ?>
-            <p>Keine Informationen vorhanden</p>
-            <?php } ?>
-        </div>
-        <?php
-    
-if (! empty($userTickets)) {
-        echo $this->partial('partials/ticket/extras.phtml', 
-                array(
-                        'tickets' => $userTickets
-                ));
-    }
-    ?>
-        <hr />
-        <h2>Teilnehmer</h2>
-        <div class="participants">
-            <p>
-        <?php
-    if (empty($participants)) {
+    if ($lan != null) {
```

Comment:
Until now we used partials only for parts which were used on different pages (in partials folder).
This would definitely increase readability but regarding above it does not make sense to make a partial out of it the way we used partials until now.
I think I will extract it into a separate partials folder (lan/partials) and include it from there